install/Microsoft.FactoryOrchestrator.CleanVolatile.service

```diff
@@ -0,0 +1,14 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+[Unit]
+Description=Microsoft Factory Orchestrator volatile file cleanup service
+
+[Service]
+Type=oneshot
+ExecStart=/bin/bash /usr/sbin/FactoryOrchestrator/Microsoft.FactoryOrchestrator.CleanVolatile.sh
+RemainAfterExit=true

```

Comment:
can we set RemainAfterExit to false.

Reply:
My understanding is I want this **true** because I only want this to run once at the **start of every OS boot**, not every time the service is started. If the user starts FO service, stop it, then starts it again, **RemainAfterExit=true** prevents the service state from being cleared. Whereas if its **false**, I'm pretty sure it will clear service state every time FO service is started, not only on first start after boot. I'll x2 check this.

Reply:
@patelp3989 confirmed that with it set to false, Microsoft.FactoryOrchestrator.CleanVolatile runs every service start not every boot. This wipes out my state file. Since I only want the state file removed after a reboot, keeping as is.